docs/config/README.md

```diff
@@ -2372,6 +2373,45 @@ symbol = " "
 style = "bold dimmed green"
 ```
 
+## Meson
+
+The `meson` module shows the current Meson developer environment status.
+
+By default the Meson project name is displayed, if `$MESON_DEVENV` is set.
+
+### Options
+
+| Option              | Default                           | Description                                                                               |
+| ------------------- | --------------------------------- | ----------------------------------------------------------------------------------------- |
+| `truncation_length` | `2^63 - 1`                        | Truncates a project name to `N` graphemes.                                                |
+| `truncation_symbol` | `"…"`                             | The symbol used to indicate a project name was truncated. You can use `""` for no symbol. |
+| `format`            | `"in [$symbol$project]($style) "` | The format for the module.                                                                |
```

Comment:
I think this module would fit a bit better on the first line along with the conda module. Please also use `via` instead of `in` to match it.